src/search-injection/constants.js

```diff
@@ -35,5 +45,11 @@ export const OPEN_OVERVIEW = 'openOverviewURL'
 export const OPEN_OPTIONS = 'openOptionsURL'
 // Storage keys
 export const HIDE_RESULTS_KEY = 'HIDE_MEMEX_RESULTS'
-export const SEARCH_INJECTION_KEY = 'SEARCH_INJECTION'
+export const SEARCH_INJECTION_KEY = 'SEARCH_INJECTION_'
```

Comment:
No. The previous `SEARCH_INJECTION` object was just a binary, the current one is a object with binaries for Google and DuckDuckGo. Different storage structures. So I stored the new one under a different name.

Reply:
I understand. So on update the user would still have the old shaped structure stored and cause issues in-code. Generally you would want to try and migrate the data (reshape from old to new) on update (`src/background`), but in this case, as it's fairly inconsequential settings data (IMO), it may be easier to just force the user back to the defaults like this (+ one-time annoyance for existing users who disabled it already). 

Although thinking about it, it should be very simple to migrate (`storage.get` + reshape + `storage.set`):
old: `boolean` => new: `{ [key: string]: boolean }`

If you want to go this route of forcing user back to default settings, I would recommend still putting a one-time `browser.storage.local.remove()` call in the "on-update" code in `src/background` to clear the old data, then use the same key (to avoid forever-forgotten mess in local storage)

@oliversauter do you have any opinions here on how it would work on upgrade if you disabled google already?